charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.1.17
+version: 1.1.18
 appVersion: "v1.11.2"
```

Comment:
We need to bump the appVersion to 1.11.3 as well, so maybe we can update that and merge this PR as part of release process.

Reply:
Synced up offline, appVersion change would be made in a separate PR.